bot/seasons/evergreen/issues.py

```diff
@@ -3,55 +3,70 @@
+                    user: str = "python-discord") -> None:
+        """Command to get issues/pull request from GitHub."""
+        url = f"https://api.github.com/repos/{user}/{repository}/issues/{str(number)}"
+        mergeURL = f"https://api.github.com/repos/{user}/{repository}/pulls/{str(number)}/merge"
 
-        if response_code in failed_status:
-            return await ctx.send(failed_status[response_code])
-
-        repo_url = f"https://github.com/{user}/{repository}"
-        issue_embed = discord.Embed(colour=Colours.bright_green)
-        issue_embed.add_field(name="Repository", value=f"[{user}/{repository}]({repo_url})", inline=False)
-        issue_embed.add_field(name="Issue Number", value=f"#{number}", inline=False)
-        issue_embed.add_field(name="Status", value=json_data["state"].title())
-        issue_embed.add_field(name="Link", value=json_data["html_url"], inline=False)
-
-        description = json_data["body"]
-        if len(description) > 1024:
-            placeholder = " [...]"
-            description = f"{description[:1024 - len(placeholder)]}{placeholder}"
+        async with self.bot.http_session.get(url) as r:
+            json_data = await r.json()
 
-        issue_embed.add_field(name="Description", value=description, inline=False)
+        if r.status in RESP_VALUE:
+            return await ctx.send(f"[{str(r.status)}] {RESP_VALUE.get(r.status)}")
+        # the original call is made to the issues API endpoint
+        # if a issue or PR exists then there will be something returned
+        # if the word 'issues' is present within the response then we can simply pull the  data we need from the
+        # return data received from the API
+        if "issues" in json_data.get("html_url"):
+            if json_data.get("state") == "open":
+                iconURL = icons.get("issue")
+            else:
+                iconURL = icons.get("issue_closed")
+        # if the word 'issues' is not contained within the returned data and there is no error code then we know that
+        # the requested data is a pr, hence to get the specifics on it we have to call the PR API endpoint allowing us
+        # to get the specific information in relation to the PR that is not provided via the issues endpoint
+        else:
+            async with self.bot.http_session.get(mergeURL) as m:
+                if json_data.get("state") == "open":
+                    iconURL = icons.get("pull-request")
+                # when the status is 204 this means that the state of the PR is merged
+                elif m.status == 204:
+                    iconURL = icons.get("merge")
+                # else by the process of elimination, the pull request has been closed
+                else:
+                    iconURL = icons.get("pull-request-closed")
 
-        await ctx.send(embed=issue_embed)
+        resp = discord.Embed(colour=0x6CC644)
```

Comment:
Please use the color from `constants.py`

Reply:
Will chuck it in tomorrow then send another RP.